package-lock.json

```diff
@@ -460,9 +460,9 @@
 			}
 		},
 		"@babel/compat-data": {
-			"version": "7.8.5",
-			"resolved": "https://registry.npmjs.org/@babel/compat-data/-/compat-data-7.8.5.tgz",
-			"integrity": "sha512-jWYUqQX/ObOhG1UiEkbH5SANsE/8oKXiQWjj7p7xgj9Zmnt//aUvyz4dBkK0HNsS8/cbyC5NmmH87VekW+mXFg==",
+			"version": "7.8.6",
```

Comment:
Can we limit this to just the uuid changes?

Reply:
Not sure how :thinking: Should I just manually remove the rest? (This was created by `npm run update-deps`.)

Reply:
`update-deps` is quite heavy handed and after recent issues I've been avoiding it.

I'd roll back the lock changes and try `npm prune` and/or `npm install`, then picking the relevant lock changes with `git add -p`.

Reply:
Done per the latest two commits. `npm prune` didn't quite work for me; I ended up following [Sérgio's instructions](https://github.com/Automattic/wp-calypso/pull/39734#issuecomment-591973316).